src/index.d.ts

```diff
@@ -13,8 +13,8 @@ declare class SpriteText extends Sprite {
   set textHeight(height: number);
   get color(): string;
   set color(color:string);
-  get backgroundColor(): string;
-  set backgroundColor(color:string);
+  get backgroundColor(): string | boolean;
```

Comment:
Thank for the change. Though it should be `string | false`, because a `true` value makes no sense. Could you update it?